src/features/loupe/FullSizeImage.jsx

```diff
@@ -192,13 +194,24 @@ const FullSizeImage = ({ image, focusIndex }) => {
             </ContextMenuItemIconLeft>
```

Comment:
Love this - let's restrict access to delete image functionality to managers though.

Reply:
:+1: Not sure if the user role I picked is the right one. Let me know

Reply:
If you don't mind adding a new one that mirrors what we have in the API that would be great: https://github.com/tnc-ca-geo/animl-api/blob/4a0f13ff910f9045475f46732b013d088109caa3/src/api/auth/roles.js#L9C52-L9C52